### PR TITLE
DevWs for Most Target Limited containing containing ZLL Dimmer Bulb

### PR DESCRIPTION
**Pull request by adurosmart**

### CODE REVIEW

**Comment by shrakuma** [comment]
@greens @MAblewiczS  can you please review this FP submission

File: devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy [hunk 1]
```diff
@@ -23,12 +23,14 @@ metadata {
 		capability "Health Check"
 
 		// Generic
-		fingerprint profileId: "C05E", deviceId: "0100", inClusters: "0006, 0008", deviceJoinName: "Generic Dimmable Light"
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 1000", outClusters: "0019"
```
Comment (greens) [inline devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy:26]: please don't change existing fingerprints

File: devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy [hunk 2]
```diff
@@ -23,12 +23,14 @@ metadata {
...
 
 		// AduroSmart
-		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FFFF, 0019", outClusters: "0019", deviceId: "0100", manufacturer: "AduroSmart Eria", model: "ZLL-DimmableLight", deviceJoinName: "Eria ZLL Dimmable Bulb"
-
-		// IKEA
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FFFF, 0019", outClusters: "0019", deviceId: "0100", manufacturer: "Trust International B.V.", model: "ZLL-DimmableLigh", deviceJoinName: "ERIA Soft White Light Bulb V2.0"
+		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FFFF, 0019", outClusters: "0019", deviceId: "0100", manufacturer: "AduroSmart Eria", model: "ZLL-DimmableLight", deviceJoinName: "ERIA Soft White Light Bulb V2.1"
+        fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FFFF, 0019", outClusters: "0019", deviceId: "0100", manufacturer: "ADUROLIGHT", model: "ZLL-DimmableLight", deviceJoinName: "ERIA Soft White Light Bulb V2.1"
```
Comment (greens) [inline devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy]: please consistently use tabs

**Comment by shrakuma** [comment]
@adurosmart can you please take a look at the above comment and make the necessary fixes. thanks

**Comment by adurosmart** [comment]
No changes to the code, we just want to change the display name of the product, so can I close the request?
@greens

**Comment by greens** [comment]
Right, so as I mentioned on the other pull requests, the reason you're seeing a lot of changes despite only adding fingerprints is because your commits were to an outdated version of the DTH. You need to rebase, or you will overwrite a significant amount of code, as you've done here.

**Pull request closed without merging**